app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  def new
+    @page_title = 'Add User'
+    @user = User.new
+  end
+
+  def create
+     @user = User.new(user_params)
+     if @user.save
```

Comment:
### [OPTIONAL] This is a tricky small bug :smile:
Here I would also log in the newly created user otherwise it won't redirect to `posts#new` the `before_action` in `posts_controller` won't allow you to pass without being logged in.
If you need more details hit me on slack